### PR TITLE
Added alt text to images on MyRoadmaps and Roadmaps

### LINKED ISSUE #81: Accessibility: give alt tags to images on Roadmaps and My Roadmaps page
### What would you like to share?

- The images on the roadmap card on roadmaps and my roadmaps page don't have an alt tag.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by giri-madhan**
## Fixes Issue

This PR fixes the following issues:

closes #81 

## Changes proposed

Here comes all the changes proposed through this PR
- Added alt texts to the images on the `Card` components on `MyRoadmaps` and `Roadmaps` components.

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: client/src/pages/MyRoadmaps/Card.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ const Card = ({ bannerImage, id, title, tagline, progress = 0 }) => {
   return (
     <Link to={`/myroadmaps/${id}`}>
       <section className="roadmap-card">
-        <img src={bannerImage} alt="" />
+        <img src={bannerImage} alt="My Roadmap banner" />
```
Comment (siddhigate) [inline client/src/pages/MyRoadmaps/Card.jsx:8]: please use the `title` as the alt tag here. The title is available through the props

File: client/src/pages/Roadmaps/Card.jsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ const Card = ({ bannerImage, id, title, tagline }) => {
   return (
     <Link to={`/roadmaps/${id}`}>
       <section className="roadmap-card">
-        <img src={bannerImage} alt="" />
+        <img src={bannerImage} alt="Roadmaps banner" />
```
Comment (siddhigate) [inline client/src/pages/Roadmaps/Card.jsx:8]: Use the title available through props as the title here.

**Review by siddhigate — APPROVED** [review]

File: client/src/pages/MyRoadmaps/Card.jsx (hunk 1 above)
Comment (giri-madhan) [reply client/src/pages/MyRoadmaps/Card.jsx]: Thank you for the review and suggestion @siddhigate! I have implemented the requested changes.

File: client/src/pages/Roadmaps/Card.jsx (hunk 2 above)
Comment (giri-madhan) [reply client/src/pages/Roadmaps/Card.jsx]: I have added the alt text from the `title` prop. Thank you.

**Comment by siddhigate** [comment]
Thanks for the PR @giri-madhan !

**Pull request merged**